platform/servicecontrol.include.md

```diff
@@ -10,6 +10,10 @@ To enable [ServiceControl](/servicecontrol) to gather this information, configur
  * configure [recoverability](/nservicebus/recoverability) to store information on messages failures;
  * [install plugins on the endpoints](/servicecontrol/plugins/) to monitor their health and sagas and use custom checks.
 
+NOTE: All endpoints should be configured to forward to the same audit, error, and ServiceControl plugin queues unless the system has been sharded between multiple ServiceControl instances.
```

Comment:
> perhaps we should plan another PR to add that option

that sounds good

I think we can leave as is, even if it doesn't explain how it works it notes there's a possibility, so if somebody is looking for something like this they'll start asking us questions